joomla/joomla-cms
# [4.0] media action resize

Removes the range slider as you cant use a range slider to enlarge see #33568 for further details

To test
apply the pr and npm i

go to media manager and confirm the sliders are gone from the resize tab and that you can still resize using the number inputs
